plugin/src/test/java/hudson/plugins/findbugs/workflow/WorkflowCompatibilityTest.java

```diff
@@ -0,0 +1,89 @@
+package hudson.plugins.findbugs.workflow;
+
+import java.io.File;
+import java.util.Set;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import hudson.FilePath;
+
+import hudson.model.Result;
+
+import hudson.plugins.analysis.util.model.FileAnnotation;
+import hudson.plugins.findbugs.FindBugsResultAction;
+import hudson.plugins.findbugs.FindBugsPublisher;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test workflow compatibility.
+ */
+public class WorkflowCompatibilityTest {
+
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
```

Comment:
Except in very specialized cases, you want to use `@Rule` with `JenkinsRule`, not `@ClassRule`. (The exception would be if you are quite sure nothing in the test modifies any state in Jenkins, which is clearly not the case here.)